### PR TITLE
New Documentation Organization

### PR DESCRIPTION
**Pull request by gbossert**
This is a complete reorg of SEKOIA.IO documentation. We probably missed some little things, that will be solved shortly.

### CODE REVIEW

**Comment by gbossert** [comment]
Besides the following minor things to fix/improve, this is far better than the existing documentation
- [ ] in XDR, usecase page "Send events to SEKOIA.IO" doesn't exist
- [ ] in XDR, the FAQ on "rsyslog debug" should be moved in the rsyslog page `xdr/features/collect/ingestion_methods/rsyslog/`

File: docs/cti/develop/rest_api/enrichments.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+---
+hide:
+  - navigation
+  - toc
+  - footer
+---
+
+!!redoc https://api.sekoia.io/v1/enricher/swagger.json?context=tip!!
```
Comment (Darkheir) [inline docs/cti/develop/rest_api/enrichments.md:8]:
```suggestion
!!redoc https://api.sekoia.io/v1/enricher/swagger.json?context=public!!
```

**Comment by goudyj** [comment]
In `XDR` > `Intakes` I dont see the example for Custom Intakes. https://docs.sekoia.io/operation_center/intakes_customformat/#practical-example
Maybe it could be added as a usecase ?

File: docs/cti/develop/rest_api/intelligence.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
+---
+hide:
+  - navigation
+  - toc
+  - footer
+---
+
+!!redoc https://api.sekoia.io/v2/inthreat/swagger.json?context=tip!!
```
Comment (Darkheir) [inline docs/cti/develop/rest_api/intelligence.md:8]:
```suggestion
!!redoc https://api.sekoia.io/v2/inthreat/swagger.json?context=public!!
```

**Comment by otetard** [comment]
> In the integration catalog, it seems [Cloudflare](https://docs.sekoia.io/operation_center/integration_catalog/cloud_and_saas/cloudflare/cloudflare/) is missing from the new organization.

Should be fixed with the latest version of this branch.

**Comment by otetard** [comment]
> In `XDR` > `Intakes` I dont see the example for Custom Intakes. https://docs.sekoia.io/operation_center/intakes_customformat/#practical-example Maybe it could be added as a usecase ?

Thanks for this feedback! Sorry, I mistakenly removed that “custom format” page. This page is now restored: https://storage.gra.cloud.ovh.net/v1/AUTH_741841e1c7da4c50a76152cbc2609bd1/sekoiaio-documentation-previews/299/xdr/features/collect/integrations/custom_format/

**Pull request merged**